08-components-lab/src/02-numberguessing.jsx

```diff
@@ -1,17 +1,34 @@
-/**
- * Lab 02:
- *
- * Implement a number guessing game component.
- *
- * The component should provide an input field for the user to guess a
- * number picked randomly by the computer (from 1 to 1,000).
- * When typing a number, the component should print "too low" or "too high" 
- * as a hint to the user.
- *
- * To make the game interesting, computer should sometimes cheat and print wrong messages.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+import React from 'react';
+
+export default class GuessNumber extends React.Component {

```

Comment:
You haven't implemented lying factor. You were asked that computer should sometimes lie.